pkg/web/web-packr.go

```diff
@@ -0,0 +1,8 @@
+// +build !skippackr
```

Comment:
Was this expected to be part of the commit?

Reply:
nope. I am fixing this.